README.md

```diff
@@ -57,12 +57,12 @@ Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 L
       - [MicroG](https://microg.org)
         - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
       - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
+    - or you can just search the web or the [xda-developers forum](https://xdaforums.com/) for an appropriate version for your device.
```

Comment:
Just a cosmetic thing: Would you mind removing the trailing slashes from all the XDA Links?